src/metabase/api/common.clj

```diff
@@ -35,6 +35,45 @@
 ;;   but it may be more instructive to look at examples in our codebase.
 ;;
 ;;  <hr />
+;;
+;; ## How does defendpoint coersion work?
+;;
+;; The `defendpoint` macro uses the `auto-coerce` function to generate a let code which binds args to their decoded
+;; values. Values are decoded by their corresponding malli schema. n.b.: Only in the arg->schema map will NOT be
```

Comment:
I might word it like this:

"n.b.: Only symbols in the arg->schema map will be coerced; additional aliases (eg. after the `:as` key) will not automatically be coerced."

Reply:
fixed